src/clang.rs

```diff
@@ -793,9 +793,13 @@ impl Comment {
     }
 
     /// Get this comment's `idx`th child comment
-    pub fn get_child(&self, idx: c_uint) -> Comment {
-        unsafe {
-            Comment { x: clang_Comment_getChild(self.x, idx) }
+    pub fn get_child(&self, idx: c_uint) -> Option<Comment> {
+        if idx  >= self.num_children() {
+            None
+        } else {
+            unsafe {
+                Some(Comment { x: clang_Comment_getChild(self.x, idx) })
+            }
         }
```

Comment:
Yeah, ideally, your later example is preferable. In this case it's not a big deal though, since there's no extra potentially unsafe stuff going on in that block.